_content/essays/recognizing-the-dhamma_geoff.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Recognizing the dhamma: a study guide"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
```

Comment:
Use the proper share url: https://www.dhammatalks.org/ebook_index.html#RecognizingTheDhamma

Also: add the pdf and epub drive_links